src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts

```diff
@@ -90,11 +67,11 @@ export class HaDeviceInfoZWaveJS extends LitElement {
       <h4>
         ${this.hass.localize("ui.panel.config.zwave_js.device_info.zwave_info")}
       </h4>
-      ${this._multipleConfigEntries
```

Comment:
The idea of this was that we only show the source if you have multiple networks.

Reply:
ah, didn't look at this closely enough. Reverted